internal/actor/lpa_progress_tracker_test.go

```diff
@@ -0,0 +1,393 @@
+package actor
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/actor/actoruid"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/date"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/identity"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/localize"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestProgressTrackerProgress(t *testing.T) {
+	lpaSignedAt := time.Now()
+	uid1 := actoruid.New()
+	uid2 := actoruid.New()
+	initialProgress := Progress{
+		Paid:                      ProgressTask{State: TaskNotStarted, Label: ""},
+		ConfirmedID:               ProgressTask{State: TaskNotStarted, Label: ""},
+		DonorSigned:               ProgressTask{State: TaskInProgress, Label: "You’ve signed your LPA"},
+		CertificateProviderSigned: ProgressTask{State: TaskNotStarted, Label: "Your certificate provider has provided their certificate"},
+		AttorneysSigned:           ProgressTask{State: TaskNotStarted, Label: "Your attorney has signed your LPA"},
+		LpaSubmitted:              ProgressTask{State: TaskNotStarted, Label: "We have received your LPA"},
+		StatutoryWaitingPeriod:    ProgressTask{State: TaskNotStarted, Label: "Your 4-week waiting period has started"},
+		LpaRegistered:             ProgressTask{State: TaskNotStarted, Label: "Your LPA has been registered"},
+	}
+	bundle, err := localize.NewBundle("../../lang/en.json")
```

Comment:
I'd much prefer to have this mocked, or weblate changes could break these tests

Reply:
Ah yeah good point 👍